src/components/Footer.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Footer = () => (
+  <div id="footer">
+    <p>
+      Need to reach us? Try <a href="https://searchworks.stanford.edu/feedback">SearchWorks feedback</a> or <a href="https://library.stanford.edu/ask/email/feedback">Library website feedback.</a>
+    </p>
```

Comment:
So uhhh ESLint really hates this. Is there some JSX-y way I should be doing this?

Reply:
Does something like this work?

```jsx
Need to reach us? Try 
<a href="https://searchworks.stanford.edu/feedback">
  SearchWorks feedback
</a> 
 or 
<a href="https://library.stanford.edu/ask/email/feedback">
  Library website feedback.
</a>
```
If not, I'd be fine w/ ignoring that rule with an inline exception